russian/girlfriend-pinch-you/go_to_the_garden/call.md

```diff
@@ -0,0 +1 @@
+Как чудесно в жаркий полдень в летнем саду! Солнце золотит все вокруг, пробивается сквозь густую зеленую листву деревьев. Ветви яблони склоняются под тяжестью краснощеких плодов. Наливаются сочные груши.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.